services/gateway/apis/http-version3/methods/transactionsDryRun.js

```diff
@@ -24,7 +23,8 @@ module.exports = {
 	rpcMethod: 'post.transactions.dryrun',
 	tags: ['Transactions'],
 	params: {
-		transaction: { optional: false, type: 'string', min: 1, pattern: regex.TRANSACTION },
+		transaction: { optional: false, type: 'any' },
```

Comment:
Let us enforce strict type checks in the gateway. It can remain `any` within the other microservices.
Please refer the following:
- https://github.com/icebob/fastest-validator#multiple-validators
- https://github.com/icebob/fastest-validator#object